app/views/workflows/index.xml.builder

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 xml.workflows(objectId: params[:druid]) do
-  @processes.each do |datastream|
-    xml.workflow(repository: params[:repo], objectId: params[:druid], id: datastream[0]) do
-      datastream[1].each do |workflow|
-        workflow.as_process(xml)
+  @workflow_steps.each do |workflow_step|
+    workflow = workflow_step[0]
```

Comment:
While we don't need these variables, I added them to give names to things, thereby clarifying their purpose.